dev-guide/cluster-fleet-evaluation.md

```diff
@@ -37,6 +43,8 @@ you are expecting, it will indicate that the change would have affected that
 cluster. However, if the _ConditionStatus_ is _false_, or it's _true_ but the
 _Reason_ does not include the _Reason_ you are expecting, the change doesn't
 affect the cluster.  You can join multiple _Reason_ to test multiple scenarios.
+This approach will get the cluster state into the in-cluster `cluster_operator_conditions` Prometheus metric, which is already exported through Telemetry.
+It will also get the cluster state into Insights, which is already collecting ClusterOperator resources.
```

Comment:
I don't think the Insights is collecting all ClusterOperator condition statuses. Please see [CCXDEV-7979](https://issues.redhat.com/browse/CCXDEV-7979)

Reply:
I think it does, e.g. see [these docs][1]:

> collects all the ClusterOperators definitions and their resources.

and [this code][2].

[1]: https://github.com/openshift/insights-operator/blob/17a1052b6da26ab1d6806f9566677b3f785bb7b6/docs/gathered-data.md#clusteroperators
[2]: https://github.com/openshift/insights-operator/blob/17a1052b6da26ab1d6806f9566677b3f785bb7b6/pkg/gatherers/clusterconfig/operators.go#L74-L127

Reply:
OK, I am not sure why the Insights team thought that all operator conditions were not collected and did that evaluation.